lib/pure/options.nim

```diff
@@ -215,6 +215,21 @@ proc get*[T](self: Option[T], otherwise: T): T {.inline.} =
   else:
     otherwise
 
+proc get*[T](self: Option[T], val: var T): bool {.inline.} =
```

Comment:
Please  rename it to `unpack` (for example), name `get` creates confusion with
```nim
proc get*[T](self: Option[T], otherwise: T): T {.inline.}
proc get*[T](self: Option[T]): lent T {.inline.}
```

Reply:
```suggestion
proc unpack*[T](self: Option[T], val: out T): bool {.inline.} =
```

Reply:
I assumed that there was one name for all procedures that fetch the data of an `Option`. It makes sense, honestly. But yeah, I can see how that can be confusing. Also, according to the Nim experimental docs, `out` needs to ensure that the value is always assigned. That makes the boolean return type completely useless, which was the whole point of this proc.